sdk/spring-sdk/src/test/java/kalix/springsdk/testmodels/eventsourcedentity/EventSourcedEntitiesTestModels.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.springsdk.testmodels.eventsourcedentity;
+
+import kalix.javasdk.eventsourcedentity.EventSourcedEntity;
+import kalix.springsdk.annotations.Entity;
+import kalix.springsdk.annotations.EventHandler;
+
+public class EventSourcedEntitiesTestModels {
+
+    @Entity(entityKey = "id", entityType = "counter")
+    public static class WellAnnotatedEntity extends EventSourcedEntity<Integer> {
+
+        @EventHandler
+        public Integer receiveStringEvent(String event) {
+            return 0;
+        }
+
+        @EventHandler
+        public Integer receivedIntegerEvent(Integer event) {
+            return 0;
+        }
+
+        public Integer publicMethodSimilarSignature(Integer event) {
+            return 0;
+        }
+
+        private Integer privateMethodSimilarSignature(Integer event) {
+            return 0;
+        }
+    }
```

Comment:
Highlighting this one as an example of the syntax in use for annotating a class and its event handlers